source/glkernel/include/glkernel/noise.h

```diff
@@ -35,6 +35,16 @@ void normal(tkernel<V<T, P>> & kernel, T mean, T stddev);
 template <typename T, glm::precision P, template<typename, glm::precision> class V>
 void normal(tkernel<V<T, P>> & kernel, const V<T, P> & mean, const V<T, P> & stddev);
 
+enum class PerlinNoiseType { Standard, Cloud, CloudAbs, Wood, Paper };
```

Comment:
Please extract into ```PerinNoiseType.h```, add explicit underlying type and format it using multiple lines of code.